_includes/indexes/hnsw-how.mdx

```diff
@@ -0,0 +1,3 @@
+HNSW indexes build a multi-layered object graph. The graph structure and HNSW algorithm result in efficient, approximate nearest neighbor [(ANN)](https://en.wikipedia.org/wiki/Nearest_neighbor_search) searches.
+
+The index and graph structure are stored in RAM memory. This makes HNSW indexes fast, but RAM is an expensive resource. Consider using [compression](/developers/weaviate/starter-guides/managing-resources/compression) to reduce the size of for your HNSW indexes.
```

Comment:
I think this should mention that the actual vectors are loaded into RAM for speed, and that's the biggest reason for the memory footprint.